### DISCUSSION TITLE
Some weird behavior in the metric system

### DISCUSSION THREAD

**Discussion by TheClaudeQC** [OpenRCT2/OpenRCT2#19807] [Q&A]
it seem that up to 120m height, everything work as 1,5m steps, but after this, the scaling become weird, it jump by 4m between the one previous to 120m to the next (118m [probably 118,5m] to 122m [121,5m would be consistent]), the one that would be like 0, 3, 6, ... start the same weird behavior at the 140m mark [139,5m In The Consistent Rule], which give 142m [141m ITCR], a bit more unusual compared to the first 120m that have the consistent height change. It become weirder again, at 160m [159m ITCR], then next on the game's scale is 162m [160,5m ITCR].

This is the image of the behavior:
![Parc sans nom 2023-04-02 23-09-47](https://user-images.githubusercontent.com/83843734/229404322-4f7c90af-451a-4761-ac3b-23e8ca8b3ee2.png)

the next one is right after 180m [178,5m ITCR], which is 182m (previous is 178m [177m ITCR]) [would be the actuall 180m ITCR], but it is hard to render, as the render at this height become chaotic. Which seem normal to break there, as it is impossible to 1x1 track pieces to reach 182m

Imperial is very consistent, whit a 5 feet change each unit (not tile inspector units)

**Comment by duncanspumpkin** [comment]
The height in meters is an approximation it's to be expected that it will not be exactly 1.5 for each unit eventually the error will accumulate enough that it won't follow a 1.5m approximation. The only one you can expect to be consistent is the actual undimensoned height units.
